README.rst

```diff
@@ -226,8 +226,8 @@ output folder (by default in ``MVS_outputs/``):
 
     mvs_tool -png
 
-post-processing
----------------
+Generate a report from simulation's output results
```

Comment:
I feel like https://github.com/rl-institut/multi-vector-simulator/blob/04a73a14358bb3ec18d089b75a236eb5ebb25f56/README.rst#generate-pdf-report-or-an-app-in-your-browser-to-visualise-the-results-of-the-simulation and https://github.com/rl-institut/multi-vector-simulator/blob/04a73a14358bb3ec18d089b75a236eb5ebb25f56/README.rst#generate-a-report-after-running-a-simulation and maybe https://github.com/rl-institut/multi-vector-simulator/blob/04a73a14358bb3ec18d089b75a236eb5ebb25f56/README.rst#generate-only-the-figures-of-a-simulations-results could be merget into one single headline. Somehow the view and navigation of this confuses me, as it looks like identical section types at the frist glance:
![grafik](https://user-images.githubusercontent.com/44204527/116379535-635fcd80-a813-11eb-95b5-8ea0feca4406.png)

Reply:
I did it in dfabaf97bb9d24f0e88f759cab767812703ac4ac